avahi-python/avahi-bookmarks.in

```diff
@@ -27,9 +27,14 @@ except ImportError:
     sys.exit(1)
 
 try:
-    import dbus.glib
-except ImportError:
-    pass
+    from dbus import DBusException
```

Comment:
I don't think this line should be necessary: I don't see any references to `DBusException` in this file.

Reply:
okay, removed. Maybe it were used before or it is just sanity check for working import.